docset/windows/addsadministration/Set-ADDefaultDomainPasswordPolicy.md

```diff
@@ -187,7 +187,7 @@ Accept wildcard characters: False
 Specifies the length of time that an account is locked after the number of failed login attempts exceeds the lockout threshold.
 You cannot login to an account that is locked until the lockout duration time period has expired.
 This parameter sets the **lockoutDuration** property of a password policy object.
-The LDAP display name (**ldapDisplayName**) of this property is msDS-LockoutDuration.
```

Comment:
It is likely better to leave the "msDS-" as originally presented given it is the name of the attribute and is easily searchable if you want to know more information about it.

Reply:
https://docs.microsoft.com/windows/win32/adschema/a-lockoutduration
https://docs.microsoft.com/windows/win32/adschema/a-msds-lockoutduration

This is confusing... Which one is the real one?

Reply:
It seems _lockoutduration_ and _ms-ds-lockoutduration_ belong to different classes.
_ms-ds-lockoutduration_ belongs to _ms-DS-Password-Settings_ the schema articles that you shared confirms that, and it is to be found in the Password Settings Container of a W2008 Functional Level.

What we need to know is to confirm which one is affected by this cmdlet, can you run this cmdlet in your lab to check ?